en/deploy/README.md

```diff
@@ -48,7 +48,7 @@ db.sqlite3
 
 And save it as `.gitignore` in the "djangogirls" folder.
 
-> **Note** The dot at the beginning of the file name is important!  If you're having any difficulty creating it (Macs don't like you to create files that begin with a dot via the Finder, for example), then use the "Save As" feature in your editor; it's bulletproof. And be sure not to add `.txt`, `.py`, or any other extension to the file name -- it will only be recognized by Git if the name is just `.gitignore`.
+> **Note** The dot at the beginning of the file name is important!  If you're having any difficulty creating it (Macs don't like you to create files that begin with a dot via the Finder, for example), then use the "Save As" feature in your editor; it's bulletproof. And be sure not to add `.txt`, `.py`, or any other extension to the file name -- it will only be recognized by Git if the name is just `.gitignore`. The .gitignore file is known as a 'dotfile' and you won't be able to see the file with normal `ls` command! Instead use `ls -a` to see the `.gitignore` file.
```

Comment:
> The .gitignore file is known as a 'dotfile'

Is that categorization relevant to workshop participants? While we don't want to withhold essential information from them, we also don't want to overload them with unnecessary jargon.

Maybe a more useful statement (for most participants) is that Linux and MacOS (or for that matter, all "Unix-like systems"; but that's another piece of jargon we avoid) consider files whose name starts with a dot hidden.

Reply:
It happened with my friend on Mac, he wasn't able to see the .gitignore file and kept on creating more!

Reply:
Well a line like - "If you're not able to find the .gitignore file in MacOS and Linux, it's because they are hidden files and to find them use the `ls -a' command!" will solve the common problem!